scripts/task_process/FTS_Transfers.py

```diff
@@ -557,12 +560,12 @@ def algorithm():
     """
 
     # TODO: pass by configuration
-    fts = HTTPRequests('fts3.cern.ch:8446/',
+    fts = HTTPRequests(FTS_ENDPOINT,
```

Comment:
does it work to pass https://fts3... here, where old code has only fts3... w/o the https:// suffix ?